CHANGES.md

```diff
@@ -144,7 +164,7 @@ The .NET Standard 2.0 version of NUnit continues to gain more functionality that
 is found in the .NET 4.5 version of the framework like setting the ApartmentState
 and enabling Timeout on tests.
 
-#### Issues Resolved
+### Issues Resolved
```

Comment:
I think that if we start using `###` for `Issues Resolved` and the other sections on the same level, then we should also change all the heading to `##`. E.g. `### NUnit 3.12 - May 14, 2019` => `## NUnit 3.12 - May 14, 2019` (and all the other headers below this one).

Reply:
I didn't even look at these older ones so I missed that.